crates/libtiny_tui/src/tui.rs

```diff
@@ -108,6 +108,9 @@ pub struct TUI {
 
     /// TabConfig settings loaded from config file
     tab_configs: TabConfigs,
+
+    /// List of blocked users
+    blocked_users: Vec<String>,
```

Comment:
I think it would be better to use a `HashSet<String>` here to have the O(1) lookup

Reply:
Done